tedana/stats.py

```diff
@@ -74,10 +74,15 @@ def computefeats2(data, mmix, mask=None, normalize=True):
     # demean masked data
     if mask is not None:
         data = data[mask, ...]
+    # normalize data (minus mean and divide by std) in the last dimension
+    # so that least-squares estimates represent correlation values (data_R)
```

Comment:
I'd say they "approximate" correlation values, because this conversion isn't technically valid as far as I can tell.